pkgs/applications/editors/vim/plugins/non-generated/vim-pony/default.nix

```diff
@@ -0,0 +1,19 @@
+{
+  fetchFromGitHub,
+  nix-update-script,
+  vimUtils,
+}:
+# The GitHub repository returns 404, which breaks the update script
+vimUtils.buildVimPlugin {
+  pname = "vim-pony";
+  version = "2018-07-27";
+  src = fetchFromGitHub {
+    owner = "jakwings";
+    repo = "vim-pony";
+    rev = "b26f01a869000b73b80dceabd725d91bfe175b75";
+    sha256 = "0if8g94m3xmpda80byfxs649w2is9ah1k8v3028nblan73zlc8x8";
```

Comment:
unreachable .  drop it

Reply:
map to https://github.com/jmcomets/vim-pony/ ?

Reply:
I dont know what vim-pony did but if they are the same why not.